include/LeMonADE/updater/moves/MoveAddScMonomer.h

```diff
@@ -96,7 +73,7 @@ template <class IngredientsType>
 void MoveAddScMonomer::init(const IngredientsType& ing)
 {
     this->resetProbability();
-    particleIndex=ing.getMolecules().size();
+    this->setParticleIndex(ing.getMolecules().size());
```

Comment:
This can be highly dangerous to deal the an index of an array exceeding its boundary. But I think most of the Features and their checkMove() never use this index of for adressing? Only the applyMove() use this, nor?